CeeFM/Decideify
# Editing Suggestion (aka changing isRecommended from null to true or false) Now Live

**Description**
As the title suggests, buttons for changing a suggestion's isRecommended property from null to true or false is now live and working. Want to further implement functionality to remove a recommendation from a suggestion and make this property null again, but will focus on that once I've hit MVP in other needed areas first. For now, changing from null to true or false, and being able to switch from true and false is good enough!

**Type of Change**


**Testing Instructions**

- Clone this project’s repository to your machine
  -If you’ve already done this and are currently working on a different branch of the project, be sure to `git add` and `git commit` whatever you’re working on on your branch, before you attempt to switch branches
  -Then `git checkout main` to switch to the project’s main branch)
- Navigate to the root directory for this repo (yourParentFolderName/Decideify)
- `git fetch -a` to retrieve the current remote-tracked branches for the project, you should see a branch named `cm-suggestions` as one of the available branches
- `git checkout cm-suggestions` to switch to my branch
- If you don't see the most recent version of my files, `git pull origin cm-suggestions`
- `Start Decideify.sln` to open the project in Visual Studio
- If you haven’t created the Decideify database, please do so in Visual Studio and use the following SQL scripts to populate it accordingly:
  - Start a new query in your database and copy in this SQL script before running to fill out tables and columns: [First SQL Script for Creating DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Create_DB.sql)
  - ctrl + a and delete what is in that query and replace it with this SQL script before running to seed the database: [Second SQL Script for Seeding DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Seed_Users_DB.sql)

- Click the execute with debugger solid green button to launch the app
- Log in as coolin@gmail.com with password thistbh
- On every content hub page and on the mylibrary.js component / page, you should see buttons that say YES and NO under a header that says "Recommend"?
- Clicking Yes will change the suggestion's isRecommended property to true, clicking No will change the suggestion's isRecommended property to false

Checklist
